near/kycdao-ntnft/src/lib.rs

```diff
@@ -152,21 +253,35 @@ impl KycdaoNTNFT {
         // Get prefilled metadata, also remove digest so it cannot be used again
         let metadata = self.authorized_token_metadata.remove(&digest).expect("Unauthorized code");
         let status = self.authorized_statuses.remove(&digest).unwrap_or_default();
+        let tier = self.authorized_tiers.remove(&digest).unwrap_or("".to_string());
```

Comment:
If we do use defaults here, maybe the default should be 'KYC_1'?

Reply:
good idea, thx